core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -159,7 +159,11 @@ public final void setAmqpHostname(final String amqpHostname) {
      * The identifier consists of the name returned by {@link #getName()} plus a UUID.
      *
      * @return The identifier.
+     * @deprecated The preferred way to obtain the <em>container-id</em> for the AMQP connection is now via
+     *             the connection factory, so that not necessarily all connections configured with this properties
+     *             object get the same id, but rather only all connections of the same connection factory instance do.
      */
+    @Deprecated
```

Comment:
Is there a particular reason why you do not want to simply remove this method?

Reply:
This is mainly for merging this commit as-is in 1.2.2. For 1.3, this method can then be removed, yes.